packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/RNTesterApplication.kt

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react.uiapp
+
+import android.app.Application
+import com.facebook.fbreact.specs.SampleLegacyModule
+import com.facebook.fbreact.specs.SampleTurboModule
+import com.facebook.react.JSEngineResolutionAlgorithm
+import com.facebook.react.ReactApplication
+import com.facebook.react.ReactNativeHost
+import com.facebook.react.ReactPackage
+import com.facebook.react.TurboReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.common.annotations.UnstableReactNativeAPI
+import com.facebook.react.common.assets.ReactFontManager
+import com.facebook.react.common.mapbuffer.ReadableMapBuffer
+import com.facebook.react.config.ReactFeatureFlags
+import com.facebook.react.defaults.DefaultComponentsRegistry.Companion.register
+import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint.load
+import com.facebook.react.defaults.DefaultReactNativeHost
+import com.facebook.react.fabric.ComponentFactory
+import com.facebook.react.flipper.ReactNativeFlipper.initializeFlipper
+import com.facebook.react.interfaces.ReactHost
+import com.facebook.react.interfaces.exceptionmanager.ReactJsExceptionHandler
+import com.facebook.react.module.model.ReactModuleInfo
+import com.facebook.react.module.model.ReactModuleInfoProvider
+import com.facebook.react.runtime.ReactHostImpl
+import com.facebook.react.shell.MainReactPackage
+import com.facebook.react.uiapp.component.MyLegacyViewManager
+import com.facebook.react.uiapp.component.MyNativeViewManager
+import com.facebook.react.uimanager.ViewManager
+import com.facebook.soloader.SoLoader
+
+class RNTesterApplication : Application(), ReactApplication {
+    private var mReactHost: ReactHostImpl? = null
+    private val mReactNativeHost: ReactNativeHost = object : DefaultReactNativeHost(this) {
```

Comment:
I have changed that, and since reactNativeHost property exists already, I've decided to make it computed, otherwise I can refactor and change `mReactNativeHost` to something like `reactNativeHostInstance`, please let me know what you think

Reply:
Here we could use the `by lazy {}` delegated property:
https://kotlinlang.org/docs/delegated-properties.html#lazy-properties